rpc_jobs/multi_node_aio_leapfrog.yml

```diff
@@ -1,69 +0,0 @@
-- project:
-    name: 'Multi-Node-AIO-Leapfrog-Testing'
-    series:
-      - newton-kilo:
-          UPGRADE_FROM_REF: "kilo"
-          branch: "newton"
-          branches: "newton"
-          KIBANA_SELENIUM_BRANCH: "newton"
-          SERIES_USER_VARS: |
-            tempest_test_sets: 'all'
-            lxc_container_vg_name: "USE_DIR_NOT_VG"
-            glance_default_store: "swift"
-            neutron_legacy_ha_tool_enabled: true
-          DEPLOY_TELEGRAF: "yes"
-      - r14.2.0-kilo:
-          UPGRADE_FROM_REF: "kilo"
-          branch: "r14.2.0"
-          branches: "r14.2.0"
-          KIBANA_SELENIUM_BRANCH: "newton"
-          SERIES_USER_VARS: |
-            tempest_test_sets: 'all'
-            lxc_container_vg_name: "USE_DIR_NOT_VG"
-            glance_default_store: "swift"
-            neutron_legacy_ha_tool_enabled: true
-          DEPLOY_TELEGRAF: "yes"
-      - newton-liberty:
-          UPGRADE_FROM_REF: "liberty"
-          branch: "newton"
-          branches: "newton"
-          KIBANA_SELENIUM_BRANCH: "newton"
-          SERIES_USER_VARS: |
-            tempest_test_sets: 'all'
-            lxc_container_vg_name: "USE_DIR_NOT_VG"
-            glance_default_store: "swift"
-            neutron_legacy_ha_tool_enabled: true
-          DEPLOY_TELEGRAF: "yes"
```

Comment:
rpc_upgrades.yml only has a single MNAIO job at present, are we ready to yank all these jobs?

Reply:
Yeah, never really had any of them work reliably so I'm ok pulling them now as they are really just wasting resources.  I'm working on the new MNAIO jobs this week which will live in rpc-upgrades.